packages/cli/src/cli/cogify/action.make.cog.ts

```diff
@@ -85,7 +85,15 @@ export class CommandMakeCog extends CommandLineAction {
     if (name == null) name = source.split('/').filter(Boolean).pop();
     if (name == null) throw new Error('Failed to find imagery set name');
 
-    const tileMatrix: string[] = this.tileMatrix.value ? [this.tileMatrix.value] : ['NZTM2000Quad', 'WebMercatorQuad'];
+    const tileMatrix: string[] = [];
+    if (this.tileMatrix.value && this.tileMatrix.value === 'All') {
```

Comment:
how about a comma separatored list here as all could be interesting?

Reply:
Do you mean still passing a string like '"['NZTM2000Quad', 'WebMercatorQuad']" from argo?

Reply:
I think it's much easier to make this `All`, because in argo we can use enum which makes a drop down list might be easier to use? However, I haven't found a multi-choice option with a separator for this in argo.  

![image](https://user-images.githubusercontent.com/12163920/190525020-6f3c231c-3352-41b5-9aa3-79127f9ab7df.png)

Reply:
Updated to support comma separated string and leave the argo workflow as default value of 'NZTM2000Quad,WebMercatorQuad'

Reply:
sorry, delayed Response how about `NZTM2000Quad/WebMercatorQuad` as there are lot more tile matrixes than just NZTM/WebMercator

Reply:
Updated. Thanks.